localization_manager/src/localization_manager_node.cpp

```diff
@@ -117,6 +126,37 @@ namespace localization_manager
         state_pub_->publish(msg);
     }
 
+    rcl_interfaces::msg::SetParametersResult Node::parameter_update_callback(const std::vector<rclcpp::Parameter> &parameters)
+    {
+        auto error = update_params<double>(
+        {{"pose_pub_rate", config_.pose_pub_rate},
+        {"fitness_score_degraded_threshold", config_.fitness_score_degraded_threshold},
+        {"fitness_score_fault_threshold", config_.fitness_score_fault_threshold},
+        {"ndt_frequency_degraded_threshold", config_.ndt_frequency_degraded_threshold},
+        {"ndt_frequency_fault_threshold", config_.ndt_frequency_fault_threshold},
+        {"x_offset", config_.x_offset},
+        {"y_offset", config_.y_offset},
+        {"z_offset", config_.z_offset}}, parameters);
+        
+        auto error_2 = update_params<int>(
+        {{"auto_initialization_timeout", config_.auto_initialization_timeout},
+        {"gnss_only_operation_timeout", config_.gnss_only_operation_timeout},
+        {"gnss_data_timeout", config_.gnss_data_timeout},
+        {"sequential_timesteps_until_gps_operation", config_.sequential_timesteps_until_gps_operation},
+        {"localization_mode", config_.localization_mode}}, parameters);
```

Comment:
I don't think we can guarantee the state machine works if the localization mode is switch mid run. Probably best to remove from this callback.

Reply:
removed it